internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -847,6 +856,25 @@ func generateBackupJobSpecIntent(ctx context.Context, postgresCluster *v1beta1.P
 				RestartPolicy:      corev1.RestartPolicyNever,
 				SecurityContext:    initialize.PodSecurityContext(),
 				ServiceAccountName: serviceAccountName,
+				// K8SPG-613
+				Volumes: []corev1.Volume{
+					{
+						Name: pNaming.CrunchyBinVolumeName,
+						VolumeSource: corev1.VolumeSource{
+							EmptyDir: &corev1.EmptyDirVolumeSource{},
+						},
+					},
+				},
+				// K8SPG-613
+				InitContainers: []corev1.Container{
+					k8s.InitContainer(
+						"pgbackrest",
```

Comment:
Maybe we should use `naming.PGBackRestRepoContainerName`?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1098/commits/018718a775e95405ed19092f7d2396fcf01cd7db